src/layouts/InsightsLayout/index.vue

```diff
@@ -50,6 +50,8 @@ export default {
   }
 
   &__insights {
+    height: calc(100vh - 88px);
```

Comment:
To facilitate maintenance, it would be better to transform "88px" into a variable and name it based on the meaning of "88px";

Reply:
Good idea, I made this correction